app/assets/javascripts/discourse/app/templates/activate-account.gjs

```diff
@@ -0,0 +1,135 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { service } from "@ember/service";
+import RouteTemplate from "ember-route-template";
+import DButton from "discourse/components/d-button";
+import hideApplicationHeaderButtons from "discourse/helpers/hide-application-header-buttons";
+import hideApplicationSidebar from "discourse/helpers/hide-application-sidebar";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import { wavingHandURL } from "discourse/lib/waving-hand-url";
+import i18n from "discourse-common/helpers/i18n";
+
+export default RouteTemplate(
+  class extends Component {
+    @service siteSettings;
+
+    @tracked accountActivated = false;
+    @tracked isLoading = false;
+    @tracked needsApproval = false;
+    @tracked errorMessage = null;
+
+    get siteName() {
+      return this.siteSettings.title;
+    }
+
+    get translatedButtonLabel() {
+      return i18n("user.activate_account.continue_button", {
+        site_name: this.siteName,
+      });
+    }
+
```

Comment:
Okay, one final suggestion - to honor all the olympic golfers 🤪 

you can inline `{{i18n "user.activate_account.continue_button" site_name=this.siteSettings.title}}` (since it's just a i18n just like any other) and also plop in `this.siteSettings.title` in the other place that uses the site title.

🏌️

```suggestion
```

(but approving now, since it looks great already!)